.github/workflows/deployDev.yml

```diff
@@ -18,28 +31,47 @@ jobs:
       REACT_APP_VERSION: 'latest'
       REACT_APP_GOOGLE_ANALYTICS_ID: ${{ secrets.GOOGLE_ANALYTICS_ID }}
       REACT_APP_GRAASP_DOMAIN: ${{ secrets.DEV_GRAASP_DOMAIN }}
+    
+    # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-      - name: Checkout
+      - name: Check out code
         uses: actions/checkout@v2
 
-      - name: Yarn Set Up
+      - name: Setup node
+        id: setup-node
         uses: actions/setup-node@v2
-      - run: yarn install
-      - run: yarn build
+        with:
+          node-version: '14'
+          check-latest: true
+
+      - name: Create .env.dev file
+        run: |
+          touch .env.dev
+          echo REACT_APP_GRAASP_DOMAIN=dev.graasp >> .env.dev
+          echo REACT_APP_GRAASP_APP_ID=${{ secrets.APP_ID }} >> .env.dev
+          echo REACT_APP_MOCK_API=false >> .env.dev
+          cat .env.dev
+
+      - name: Yarn build dev
+        id: build-image
+        run: |
+          yarn install
+          yarn build:dev 
 
       - name: Configure AWS Credentials
+        id: configure-aws
         uses: aws-actions/configure-aws-credentials@v1
         with:
           aws-access-key-id: ${{ secrets.APPS_AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.APPS_AWS_SECRET_ACCESS_KEY }}
           aws-region: ${{ secrets.APPS_AWS_REGION }}
 
-      - name: Copy files to AWS Bucket
+      - name: S3 Sync
+        id: sync-s3
         env:
-          APP_DIR: '${{ secrets.APPS_DEV_BUCKET }}/${{ secrets.APPS_DEVELOPER_ID }}/${{ secrets.APP_ID }}/${{env.VERSION}}/'
-        run: aws s3 sync ${{env.BUILD_FOLDER}} s3://${APP_DIR} --delete
+          APP_DIR: '${{ secrets.APPS_DEV_BUCKET }}/${{ secrets.APP_ID }}/'
```

Comment:
As mentioned we used version. Maybe we can go with the default version `latest`?

Reply:
I checked some of the folders and saw that the files were uploaded directly to the /bucket/app_id/ folder.  We can keep the previous structure